### ISSUE TITLE
Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.

### ISSUE THREAD

**Issue by mr-mmmmore** [google/flutter.plugins#140]
Hi,

I just upgraded my app to Dart 2.17 and Flutter 3 and I now get the following warning when building:

```yml
../../../../../Flutter/.pub-cache/hosted/pub.dartlang.org/audiofileplayer-2.1.1/lib/audiofileplayer.dart:248:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../../../Flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.addObserver(this);
                   ^
../../../../../Flutter/.pub-cache/hosted/pub.dartlang.org/audiofileplayer-2.1.1/lib/audiofileplayer.dart:257:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../../../Flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.addObserver(this);
                   ^
../../../../../Flutter/.pub-cache/hosted/pub.dartlang.org/audiofileplayer-2.1.1/lib/audiofileplayer.dart:267:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../../../Flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.addObserver(this);
                   ^
../../../../../Flutter/.pub-cache/hosted/pub.dartlang.org/audiofileplayer-2.1.1/lib/audiofileplayer.dart:276:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../../../Flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.addObserver(this);
                   ^
../../../../../Flutter/.pub-cache/hosted/pub.dartlang.org/audiofileplayer-2.1.1/lib/audiofileplayer.dart:480:22: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../../../Flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance!.removeObserver(this);
                     ^
../../../../../Flutter/.pub-cache/hosted/pub.dartlang.org/audiofileplayer-2.1.1/lib/audiofileplayer.dart:705:22: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../../../Flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance!.removeObserver(playingAudio);
```

**Comment by gilescm** [comment]
FYI I created a PR to fix this issue #135 it just needs review and merge

**Comment by mr-mmmmore** [comment]
Duplicate of #135
